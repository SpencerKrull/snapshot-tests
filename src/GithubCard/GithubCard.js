import React from 'react'
import Card from 'react-bootstrap/Card'
import selfie from './selfie.jpg'

function GithubCard(){
    return (
        <div className="pic">
            <Card style={{ width: "20rem" }}>
                <Card.Img variant="top" src={selfie} />
                <Card.Body>
                    <Card.Title>Spencer Krull</Card.Title>
                    <Card.Text>Musician and software developer in training</Card.Text>
                </Card.Body>
            </Card>
        </div>
    );
}

export default GithubCard