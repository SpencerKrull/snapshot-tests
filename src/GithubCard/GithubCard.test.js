import GithubCard from './GithubCard'
import renderer from 'react-test-renderer'

test('renders a snapshot', () => {
    const tree = renderer.create(<GithubCard />).toJSON()
    expect(tree).toMatchSnapshot()
})