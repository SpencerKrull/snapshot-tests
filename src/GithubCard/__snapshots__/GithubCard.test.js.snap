// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="pic"
>
  <div
    className="card"
    style={
      Object {
        "width": "20rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="selfie.jpg"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Spencer Krull
      </div>
      <p
        className="card-text"
      >
        Musician and software developer in training
      </p>
    </div>
  </div>
</div>
`;
